projects/js-toolkit/packages/portal-base/src/start/customElement.ts

```diff
@@ -37,4 +54,108 @@ export default async function customElement(project: Project): Promise<void> {
 	const server = new webpackDevServer(devServerOptions, compiler);
 
 	await server.start();
+
+	process.on('exit', () => {
+		print(warn`
+Undeployed live development Remote App from Liferay.`);
+		print(info`
+Run 'yarn deploy' if you want to make your Remote App available in your Liferay
+server again by deploying a non-live version of it.`);
+
+		fs.rmSync(zipFile.asNative);
+	});
+}
+
+async function checkForOldZipFile(zipFile: FilePath): Promise<void> {
+	if (!fs.existsSync(zipFile.asNative)) {
+		return;
+	}
+
+	const stats = fs.statSync(zipFile.asNative);
+
+	if (Date.now() - stats.mtimeMs > 1000 * 60 * 120) {
```

Comment:
Can we set `1000 * 60 * 120` to a variable with a explicit name?

Reply:
for example, something like:

```javascript
const ONE_WEEK = 1000 * 60 * 120;
```

:trollface: